screens/HomeScreen.tsx

```diff
@@ -26,11 +29,50 @@ export default function HomeScreen({
   navigation: { navigate },
 }: HomeScreenProps) {
   const navigation = useNavigation();
+  const [user, setUser] = useState<User>();
 
+  const [companies, setCompanies] = useState<User[]>();
+
+  if (!user) {
+    backendFetch<User>('GET', 'account')
+      .then(e => {
+        setUser(e as User);
+      })
+      .catch(console.log);
+  }
```

Comment:
Wrap in a useEffect.
```ts
useEffect(() => {
  // paste this section in here
}, [])
```

Also, make a seperate file in the ``lib`` folder for ``getUser``. That makes it more portable and easier to type propelry.

Example usage code would be:
```ts
getUser().then(setUser).catch(alert)
```